src/transactions/test/MergeTests.cpp

```diff
@@ -700,15 +700,41 @@ TEST_CASE("merge", "[tx][merge]")
             };
 
         for_versions_from(14, *app, [&] {
+            uint32_t ledgerVersion;
```

Comment:
I thought it was cleaner this way instead of creating multiple lambdas and using overlapping `for_version` blocks. I'm fine with changing it though to follow our normal convention.

Reply:
That's fine, no need.